lib/git/helpers.sh

```diff
@@ -13,4 +13,11 @@ function fail_if_not_git_repo() {
     return 1
   fi
   return 0
-}
\ No newline at end of file
+}
+
+bin_path() {
+  if [[ -n ${ZSH_VERSION:-} ]];
+    then builtin whence -cp "$1" 2> /dev/null
+    else builtin type -P "$1"
+  fi
+}
```

Comment:
Aha! My previous comment seems moot, now that I see this.

Reply:
All good :)